src/main/java/com/cloudblue/connect/internal/operations/CaseOperations.java

```diff
@@ -0,0 +1,94 @@
+package com.cloudblue.connect.internal.operations;
+
+import com.cloudblue.connect.internal.operations.connections.CBCConnection;
+import com.ctc.wstx.io.EBCDICCodec;
+import com.cloudblue.connect.api.models.CBCCase;
+import com.cloudblue.connect.api.models.CBCConversationMessages;
+import com.cloudblue.connect.api.parameters.Embeddable;
+import com.cloudblue.connect.api.parameters.NewConversationMessage;
+import com.cloudblue.connect.api.clients.constants.HttpMethod;
+import com.cloudblue.connect.api.exceptions.CBCException;
+import com.cloudblue.connect.api.clients.utils.URLBuilder;
+import com.cloudblue.connect.api.parameters.NewHelpdeskCaseParameter;
+import com.cloudblue.connect.api.parameters.ChangeStatusHelpdeskCaseParameter;
+import com.cloudblue.connect.api.parameters.common.ResourceActionParameter;
+import com.fasterxml.jackson.core.type.TypeReference;
+import org.mule.runtime.extension.api.annotation.param.Connection;
+import org.mule.runtime.extension.api.annotation.param.MediaType;
+import org.mule.runtime.extension.api.annotation.param.ParameterGroup;
+import org.mule.runtime.extension.api.annotation.param.display.DisplayName;
+import org.mule.weave.v2.grammar.NotEqOpId;
+
+import static org.mule.runtime.extension.api.annotation.param.MediaType.ANY;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+public class CaseOperations {
+    
+    private final Logger LOGGER = LoggerFactory.getLogger(CaseOperations.class);
+    
+    @MediaType(value = ANY, strict = false)
+    @DisplayName("Get Helpdesk Case")
+    public CBCCase getCase(
+            @Connection CBCConnection connection,
+            @ParameterGroup(name="Case ID") ResourceActionParameter getCaseParameter
+    ) throws CBCException {
+
+        URLBuilder urlBuilder = new URLBuilder()
```

Comment:
Use "Q" instead of raw call to exchange method.

Reply:
Done